engineering/methodology.md

```diff
@@ -1,131 +1,196 @@
+* [Team Meeting](#team-meeting)
+* [Standup](#standup)
+* [Demo](#demo)
+* [Keeping track of things](#keeping-track-of-things)
+* [Definitions](#definitions)
+  * [What does TODO mean?](#what-does-todo-mean)
+  * [What does Done mean?](#what-does-done-mean)
+  * [What does Blocker mean?](#what-does-blocker-mean)
 
 ## Intro
 
-These are the notes from
- - 2016-09-12 Scrum/OKR
- - 2017-05-08 Sprint #13 Retrospective
+These are the notes from:
+
+* 2017-01-09 Scrum of Scrums
+* 2017-01-10/11 Workflow discussions
+
+Meetings for the Engineering team leads at source{d}.
+It’s the replacement for the [previous workflow](https://github.com/src-d/guide/blob/abe7932e3bc58898e49a5c1c0e3e2cac6d387144/engineering/methodology.md) \w SCRUM
+
+## OKRs
+
+Company publishes OKRs every quarter.
+
+2018Q1 OKRs in https://github.com/src-d/guide/blob/general/OKRs/README.md
+
+Each team is expected to publish it’s own ORK, aligned with the company ones.
+Engineering roles are not expected to have individual objectives.
+
+## The Big Change
+
+In 2018Q1 SCRUM is replaced by a simpler, company and team-level Kanban boards,
+so that all iterations including planning/work/retrospective do not happen
+synchronously at the same weeks for all the teams.
+
+Meetings for high-level sync including product direction, vision, etc are
+proactively scheduled by each Team Lead as needed. All Leads have periodical
+hands-on meetings \w CTO.
+
 
-meetings for the Engineering team at source{d}.
+## Kanban
 
-## OKR
+Kanban board, a tool used to visualize work and optimize the flow of the work
+among the team. (source)
 
-- Officially, engineering grunts will not have Objectives, although they can
-  create their own.  This has to do with how to mix SCRUM and OKRs properly.
-  The details about this are still not defined, @mcuadros has to talk with
-  @jorgeschnura to see how to best fit SCRUM and OKR.
+The kanban relies upon full transparency of work and real-time communication of
+capacity, therefore the kanban board should be seen as the single source of
+truth for the team's work.
 
-## SCRUM
+The kaban columns are defined as follows:
 
-- We will be having different SCRUMS per team as we are so many now.
+### [Company](https://github.com/orgs/src-d/projects/9) Kanban
 
-- Sprints will last 2 weeks.
+Column      | Description
+------------|------------
+Backlog     | Only contains Epics and user stories.
+TODO        |
+In Progress |
+Done        |
 
-- **Goal of the Sprint**: deliver something usable by the end of it.
+### Team Kanban
 
-- Tools: up to a team to decide. But backlog should be is in GH issues.
+Column           | Description
+-----------------|------------
+Backlog          | Break-down of the Epics for each product on actual tasks
+TODO             | Tasks from backlog that are immediately ready to be act up on. Grooming a backlog to update this column is expected to be done on daily basis by the Lead. See the Definition of TODO section below.
```

Comment:
s/to be act up on/to be acted on/ (or acted upon)
s/done on daily basis/done on a daily basis/
s/See the Definition of TODO section below/See the section below [What does TODO mean?](#what-does-todo-mean)/